src/sage/topology/simplicial_complex.py

```diff
@@ -4846,11 +4846,19 @@ def intersection(self, other):
             F = F + [s for s in self.faces()[k] if s in other.faces()[k]]
         return SimplicialComplex(F)
 
-    def bigraded_betti_numbers(self, base_ring=ZZ):
+    def bigraded_betti_numbers(self, base_ring=ZZ, verbose=False):
         r"""
         Return a dictionary of the bigraded Betti numbers of ``self``,
         with keys `(-a, 2b)`.
 
+        INPUT:
+
+        - ``base_ring`` -- (optional, default ``ZZ``) the base ring used
+          when computing homology
+        - ``verbose`` -- (optional, default ``False``) if ``True``, print
```

Comment:
You should add a note that this avoids the cache. Also `(optional, default: X)` -> `(default: X)`.

Reply:
Thank you.